example/example-reader.py

```diff
@@ -38,7 +38,7 @@ def reader_example():
 
     print("First 5 points:")
     for i in range(5):
-        print(node_points.Get(i))
+        print(node_points[i])
```

Comment:
I thought we could do
```
for points in node_points[:5]:
        print(points)
```
But doesn't seem to like it?

Reply:
That type of indexing might not be automatically supported?

Reply:
I have not added slicing support yet... It would require returning a new "points" object to iterate over. Although, now that I think about it, that's not a  big deal because the Points object isn't bound to a node or anything, so I can add that in. https://github.com/pybind/pybind11/blob/master/tests/test_sequences_and_iterators.cpp#L229

Reply:
Cool, do you want to do this within this PR or should I approve it and we add that to backlog?